src/ScaleUnitManagement/WorkloadSetupOrchestrator/Utilities/AOSClient.cs

```diff
@@ -13,17 +13,35 @@ namespace ScaleUnitManagement.WorkloadSetupOrchestrator.Utilities
     class AOSClient
     {
         private readonly HttpClient httpClient;
+        private readonly string requestPathPrefix = "";
 
-        private AOSClient(HttpClient httpClient)
+        private AOSClient(HttpClient httpClient, string requestPathPrefix = "")
         {
             this.httpClient = httpClient;
+            this.requestPathPrefix = requestPathPrefix;
         }
 
-        public static async Task<AOSClient> Construct(string AosResourceId, string AosEndpoint)
+        public static async Task<AOSClient> Construct(EnvironmentType environmentType, string AosResourceId, string AosEndpoint)
         {
-            string aadTenant = Config.Authority();
-            string aadClientAppId = Config.AppId();
-            string aadClientAppSecret = Config.AppSecret();
+            string aadTenant;
+            string aadClientAppId;
+            string aadClientAppSecret;
+            string requestPathPrefix = "";
+            if (environmentType == EnvironmentType.LBD)
+            {
+                aadTenant = Config.AdfsAuthority();
+                aadClientAppId = Config.AdfsAppId();
+                aadClientAppSecret = Config.AdfsAppSecret();
+                requestPathPrefix = "namespaces/AXSF/";
```

Comment:
we could get this value from Config based on the environment type, and remove the if statement completely